tsconfig.json

```diff
@@ -8,8 +8,8 @@
     "noImplicitAny": true,
     "noImplicitReturns": true,
     "noImplicitThis": true,
-    "noUnusedLocals": true,
-    "noUnusedParameters": true,
+    "noUnusedLocals": false,
+    "noUnusedParameters": false,
```

Comment:
:(

Reply:
This change converts compile errors into linter error. The good thing about having those as linter errors is that you can deactivate the linter on a case by case basis if you want to explicitly ignore it. This is especially useful during development when some parts of the code are already created but not yet connected. I changed this because I faced a case where with this compile setting, we could not make any progress:

```
  const onSignup = async (formValues: FormValues): Promise<void> => {
    // eslint-disable-next-line @typescript-eslint/no-unused-vars, no-unused-vars
    const password = formValues[PASSWORD_FIELD];
    accountName.current = formValues[ACCOUNT_NAME_FIELD];
```

But since the linter still complains, changing this TypeScript config should not lead to bad code or bugs.